controllers/topics.controllers.js

```diff
@@ -0,0 +1,14 @@
+const {readTopics} = require('../models/topics.models')
+
+const getTopics = (request, response, next) => {
+    const path = request.originalUrl;
```

Comment:
Do you need this line? We only get to this point if the url matches in the first place? So is there a point in passing the url through to the model and then checking?

It is a little redundant at this point :)